docker-vms/vxtwinio/vxtwinio.yaml

```diff
@@ -0,0 +1,26 @@
+# A vxtwinio application.
+
+# application configuration
+
+worker_name: vxtwinio_application
+transport_name: vxfreeswitch_transport
+
+web_path: "/vxtwinio"
+web_port: 8000
```

Comment:
Won't this clash with the transport on port 8000? Or are these on different machines?